src/items.rs

```diff
@@ -1528,7 +1528,7 @@ pub(crate) fn rewrite_opaque_type(
         context,
         indent,
         "type",
-        " =",
+        " = impl",
```

Comment:
This is not quite right, if the rhs consists of multiple lines then the `impl` will be orphaned:

```rust
pub type Type = impl
    VeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeryLongType<Target = i8>;
```

Reply:
Thank you the review. I fixed it.